modules/ROOT/taxonomy.adoc

```diff
@@ -181,21 +181,19 @@
   *** xref:queriable-logging.adoc[Queriable logger]
 * Identity Management
  ** Authorization
-  *** Organization
-   **** xref:organization-overview.adoc[Overview]
+  *** xref:organization-overview.adoc[Organization]
    **** xref:approaches-to-managing-organizations-and-actor-mapping.adoc[Approaches to managing organizations and actor mapping]
    **** xref:organization-management-in-bonita-bpm-studio.adoc[Organization management in Bonita Studio]
    **** xref:custom-user-information-in-bonita-bpm-studio.adoc[Custom User Information in Bonita Studio]
-  *** Profiles
-   **** xref:profiles-overview.adoc[Overview]
+  *** xref:profiles-overview.adoc[Profiles]
    **** xref:profile-creation.adoc[Profile editor]
   *** xref:rest-api-authorization.adoc[REST API authorization]
   *** xref:custom-authorization-rule-mapping.adoc[Authorization rules]
   *** xref:bdm-access-control.adoc[BDM access control]
- ** Authentication
-  *** xref:user-authentication-overview.adoc[User authentication overview]
+ ** [Authentication]
```

Comment:
I don't understand this change, and I disagree: it looks weird and doesn't make sens for users.

```suggestion
 ** Authentication
```

Reply:
I do not understand what you disagree with ; can you be more specific, and explain what you would rather have?